app/src/main/java/com/codesoom/assignment/handler/TodoHttpHandler.java

```diff
@@ -0,0 +1,62 @@
+package com.codesoom.assignment.handler;
+
+import com.codesoom.assignment.Exception.NotFoundException;
+import com.codesoom.assignment.service.TodoService;
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+
+import java.io.*;
+import java.net.URI;
+import java.util.stream.Collectors;
+
+public class TodoHttpHandler implements HttpHandler {
+    private TodoService todoService = new TodoService();
+    @Override
+    public void handle(HttpExchange exchange) throws IOException {
+        String method = exchange.getRequestMethod();
+        String content = "Hello World";
+        Long taskId = 0L;
+        URI uri = exchange.getRequestURI();
+        String path = uri.getPath();
+
+        InputStream inputStream = exchange.getRequestBody();
+        String body = new BufferedReader(new InputStreamReader(inputStream))
+                .lines()
+                .collect(Collectors.joining("\n"));
+        System.out.println(method + " " + path);
+        try{
+            if (method.equals("GET") && path.equals("/tasks")) {
+                content = todoService.getTasks();
+                exchange.sendResponseHeaders(200, content.getBytes().length);
+            } else if (method.equals("POST") && path.equals("/tasks")) {
+                content = todoService.postTask(body);
+                exchange.sendResponseHeaders(201, content.getBytes().length);
+            } else if (method.equals("PUT") && path.matches("/tasks/[0-9\\w]+")) {
+                taskId = Long.parseLong(path.split("/")[2]);
+                content = todoService.putTask(taskId, body);
+                exchange.sendResponseHeaders(200, content.getBytes().length);
+            } else if (method.equals("GET") && path.matches("/tasks/[0-9\\w]+")) {
+                taskId = Long.parseLong(path.split("/")[2]);
+                content = todoService.getTask(taskId);
+                exchange.sendResponseHeaders(200, content.getBytes().length);
+            } else if (method.equals("DELETE") && path.matches("/tasks/[0-9\\w]+")) {
+                taskId = Long.parseLong(path.split("/")[2]);
+                content = todoService.deleteTask(taskId);
+                exchange.sendResponseHeaders(204, content.getBytes().length);
+            } else {
+               throw new NotFoundException("존재하지 않는 API");
+            }
+        }catch(NotFoundException e){
```

Comment:
사용하고 있는 IDE 에서 포매터를 사용해보세요. 그러면 이런 종류의 코드 변경은 신경쓰지 않아도 되겠죠. IntelliJ 또는 Eclipse에서 어떤 키를 눌렀을 때 포매터가 작동하는지 조사해 보시고, 자동으로 작동하게 하는 방법이 없는지도 연구해 보세요.

```suggestion
        } catch(NotFoundException e) {
```

Reply:
적용했습니다! 감사합니다.